src/sass/components/_movie-modal.scss

```diff
@@ -1,9 +1,212 @@
-@use "../base/typography" as *;
+@use '../base/typography' as *;
+@use '../abstracts/mixins' as *;
+@use '../abstracts/variables' as *;
 
-.movie-lightbox {
-    display: grid;
+.modal {
+  display: grid;
 }
-.movie-lightbox__title {
-    font-size: var(--fs-lightbox-title);
-    line-height: var(--lh-lightbox-title);
+.modal__title {
+  font-size: var(--fs-lightbox-title);
+  line-height: var(--lh-lightbox-title);
+}
+.backdrop {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0px;
+  top: 0px;
+  opacity: 1;
+  background: rgba(0, 0, 0, 0.25);
+  transition: opacity 250ms linear;
+  &.is-hidden {
+    visibility: hidden;
+    opacity: 0;
+    pointer-events: none;
+  }
+}
+.movie-form {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: $bg-color-light;
+  overflow-y: hidden;
+  width: 280px;
+  height: 85%;
+  margin-top: 20px;
+  margin-bottom: 38px;
+  overflow-y: scroll;
+
+  @include mq(tablet) {
+    width: 704px;
+    height: 552px;
+    overflow: hidden;
+  }
+  @include mq(desktop) {
+    width: 806px;
+    height: 558px;
+    overflow: hidden;
+  }
+}
+.movie {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  @include mq(tablet) {
+    flex-direction: row;
+  }
+}
+.close-btn {
+  padding: 0;
+  stroke: $font-color-dark;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  height: 30px;
+  top: 8px;
+  right: 8px;
+  border: none;
+  background-color: $bg-color-light;
+  cursor: pointer;
+  &:hover,
+  &:focus {
+    stroke: $font-color-accent;
+  }
+}
+
+.movie__description {
```

Comment:
Jak movie-description na lini 80 i lini 117 to to samo to je scal w jedno

Reply:
done